.fixtures.yml

```diff
@@ -3,3 +3,5 @@ fixtures:
     facts:
       repo: "puppetlabs/ruby_task_helper"
       ref: "0.4.0"
+  repositories:
```

Comment:
I would like to see ruby_plugin_helper released and managed like the task helper here.

Reply:
A little confused on what the field for the forge repo is supposed to be -- is it just whatever name we want or something set in forge? Is this what we would want?

```
    ruby_plugin_helper:
      repo: "puppetlabs/ruby_plugin_helper"
      ref: "0.1.0"
```

Reply:
I think it is meaningless. I used to think it was the name a a symlink or but it looks like it ignores it. But yeah what you have should work once the module is released.